bin/export-db-to-sqlite.sh

```diff
@@ -7,125 +7,305 @@
 # Exports the postgres database to a sqlite file, which can be copied to the
 # cloud bucket for consumption by web pods.
 
-# Usage:
-# ./bin/export-db-to-sqlite.sh /path/to/output.db
-
 # CRITICAL: requires DATABASE_URL in the environment (or temporarily set at the
 # time of running), pointing to the source *Postgres* DB
 
+# Usage if you have DATABASE_URL in your Environment:
+# cd /path/to/checkout/of/bedrock/
+# ./bin/export-db-to-sqlite.sh /path/to/output.db
+
+# Usage if you do not have DATABASE_URL in your Environment:
+# cd /path/to/checkout/of/bedrock/
+# DATABASE_URL="postgres://user:pass@host:5432/bedrock" ./bin/export-db-to-sqlite.sh /path/to/output.db
+
+# If you want to run this in debug mode, insert `bash -x` (without quotes) before ./bin/
+
+# Set up variables to point to the new output DB and to a temporary sql script which we'll generate later
+
 output_db=$1
```

Comment:
Curious if a default path would be nice here. Like, wherever bedrock expects the sqlite db to live. Otherwise I would have to look that up each time and do some file copy locally. Would be nice to just drop it where it needs to go to work.

Reply:
I get your point, tho cos this will usually run on a clock pod in k8s, where the path may well be custom, maybe we can leave it as-is.

Reply:
yeah, didn't think at the time of writing that this would rarely, if ever, be run locally by us devs.